recipe/meta.yaml

```
@@ -21,19 +21,19 @@ requirements:
     - python >=3.7
     - pip
     - nodejs <16
-    - jupyter-packaging >=0.10,<1
+    - hatchling
```

`hatch-jupyter-builder` is also needed:

```suggestion
    - hatch-jupyter-builder >=0.3.2
    - hatchling >=1.8.1
```

---

Thanks!